packages/types/src/server.ts

```diff
@@ -90,6 +93,9 @@ export interface OpenProjectLaunchOptions {
   onChange?: WebSocketOptionsCallback
   onError?: (err: Error) => void
 
+  // Manager used to communicate Cypress lifecycle events to the protocol
```

Comment:
Suggest making it a little more generic:
```suggestion
  // Manager used to communicate with the Cloud protocol
```